### PR TITLE
Code 90 enable jsxa11y custom eslint

### PR DESCRIPTION
**Pull request by NagaPradeepGajula-osi**
Enabled Jsx-a11y plugin and some custom best practices agreed upon by all leads. Also had couple of fixes in. Do not want the PR to be messy with all fixes will get to them in other.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -15,6 +16,33 @@ module.exports = {
...
+      'error',
+      {
+        components: ['label'],
+        required: {
+          some: ['nesting', 'id'],
+        },
+        allowChildren: true,
+      },
+    ],
+    'no-console': [2],
```
Comment (eckdanny-osi) [inline .eslintrc.js]: i'd prefer a `warn` and a [babel transform](https://babeljs.io/docs/en/babel-plugin-transform-remove-console) to rake them from dist artifacts

File: .eslintrc.js [hunk 2]
```diff
@@ -15,6 +16,33 @@ module.exports = {
...
+      ignore: [-1, 0, 1, 2]
+    }],
+    'no-useless-concat': [2],
+    'no-var': [2],
+    'prefer-const': [2],
+    'prefer-template': [2],
+    'radix': [2, 'as-needed'],
+    'react/no-did-mount-set-state': [1],
+    'react/no-did-update-set-state': [1],
+    'react/display-name': 0,
```
Comment (eckdanny-osi) [inline .eslintrc.js]: let's yank `react/*` lints into a scoped PR

**Comment by NagaPradeepGajula-osi** [comment]
Sure I will make seperate Pr for the common one.

**Comment by NagaPradeepGajula-osi** [comment]
Changes are done as mentioned kept the PR scope to just jsx-a11y and some autofixes.

**Pull request merged**